circle.yml

```diff
@@ -65,17 +69,21 @@ test:
     - .circle/docker.sh test st2api 'st2 --version'
 
 deployment:
-  stable:
-    branch: /v[0-9]+(\.[0-9]+)*/
-    commands:
-      - .circle/bintray.sh deploy debian /tmp/st2-packages
-      - .circle/docker.sh deploy st2actionrunner st2api st2auth st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer
-
-deployment:
-  unstable:
-    branch: master
+  publish:
+    owner: StackStorm
+    branch:
+      - master
+      - /v[0-9]+(\.[0-9]+)*/
     commands:
-      - .circle/bintray.sh deploy debian /tmp/st2-packages
+      # Deploy to Bintray all artifacts for respective distros in parallel
+      - |
+        DISTROS=($DISTROS)
+        for i in $(seq 0 $((CIRCLE_NODE_TOTAL-1))); do
+          echo Deploying Bintray artifacts for "${DISTROS[$i]}" ...
+          .circle/bintray.sh deploy ${DISTROS[$i]}_staging ~/packages/${DISTROS[$i]} &
+        done
+        wait
+      - echo Web Hook triggering st2 Staging box with build context payload will be here
       - .circle/docker.sh deploy st2actionrunner st2api st2auth st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer
```

Comment:
Yeah, I made it looking this way to show them explicitly/more readable for me in `circle.yml` file.

Probably good to move them into global env var, same as `$DISTROS`.